survey/summary.md

```diff
@@ -0,0 +1,130 @@
+malware in releases when it happens.
+
+Rust, Ruby and Gradle have no plans on ensuring that install and build code
+doesn't allow execution of arbitrary code on the host in an unprotected way.
+This is an area that requires further investigation into why that is not a
+priority and look into a way to be able to have isolation and provenance of
+builds without this property.
+
+![image](https://user-images.githubusercontent.com/3060102/220681554-3b469808-3fdd-441e-b91c-0cf060461af2.png)
+
+## Protecting the ecosystem
+
+Most ecosystems are not currently performing any security testing against its
+code/infrastructure. However, all of them are interested (7) or have it planned
+(2). These ecosystems could be funded for security testing.
+
+All ecosystems have a process in place for vulnerability disclosure or are
+interested in doing so.
+
+Only 1 ecosystem has additional supply chain security guarantees for the
+binaries/tooling that it produces. This is an area for improvement overall.
+
+![image](https://user-images.githubusercontent.com/3060102/220682148-2d488135-4695-41b5-a6ab-e5f85883f612.png)
+
+## Auth and Credential Management
+
+Generally there is interest in protecting token security, which is positive
+feedback. There are 2 ecosystems that currently have no plans on requiring MFA.
+Further investigation on why would be helpful.
+
+## Policy:
+
+With policy, there is a mix of implementations that help provide ability to
+create policy on keys/hashes as well as checking for known vulnerabilities, as
+well as a significant number of ecosystems (4-5) which do not but are
+interested.
+
+For policies that take into account 3rd party audits and attestations, only 1
+ecosystem provided functionality. The rest are mostly interested (8) and planned
+(1). This could be somewhere that a supply chain knowledge graph project and
+public instance can be leveraged to lower the cost to do so.
+
+![image](https://user-images.githubusercontent.com/3060102/220683454-3d685cb4-861a-498c-b3b9-6fbde42b7d3a.png)
+
+## SBOMs
+
+7 of 11 ecosystems do not currently have support for generating SBOM but are
+interested (5) or planned (2). Additional tooling support is required and could
+be a place where the OpenSSF SBOMs everywhere workstream 9 can be helpful.
+
+Most SBOM formats are around SPDX and CycloneDX, with the exception of golang
+that provides an intermediary format. This is a positive indicator of
+convergence around the two SBOM standards.
+
+![image](https://user-images.githubusercontent.com/3060102/220681004-a45e2790-aa2a-45fc-8262-3bb36a183897.png)
+
+
+## Identify / produce good dependencies
+
+In general most ecosystems allow pinned dependencies that are immutably
+verifiable. However, there are 3 ecosystems that do not have this yet. Further
+investigation on why would be helpful.
+
+Only 27.3% of ecosystems have fuzzing services and utilities available, 45.5%
+have no plans on implementing and the last 27.3% are interested in exploring.
```

Comment:
These don't sum to 100%. Which is to be expected given rounding error. Perhaps at the introduction we can warn that amounts will not sum to exactly 100%?

Reply:
![image](https://user-images.githubusercontent.com/3060102/220773019-8806c04c-b689-4cb5-b71f-eb4bcb87bb78.png)


:| lol... it's one of those rounding error things... i'll reduce it to 45.4 for consistency.